presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -13,14 +13,14 @@ class AnonymizerEngine:
     and replaces the PII entities with the desired transformations.
     """
 
+    def __get_sub_classes():
+        sub_classes = {}
+        for cls in Anonymizer.__subclasses__():
+            sub_classes[cls.__name__.lower()] = cls
+        return sub_classes
+
     logger = logging.getLogger("presidio-anonymizer")
-    builtin_anonymizers = {
-        "mask": Mask,
-        "fpe": FPE,
-        "replace": Replace,
-        "hash": Hash,
-        "redact": Redact,
-    }
+    builtin_anonymizers = __get_sub_classes()
```

Comment:
Nice! 👍 
Do you think that there's a place to be more implicit, and an additional interface method, like get_display_name? In case we want to pass a different name in the API, and that the sender will be more aware of the connection between the method class and and api method?

Reply:
yes, that's a good idea. I'll do a quick update